docker-compose.yml

```diff
@@ -1,17 +1,46 @@
 version: '3.6'
 services:
-  license_finder:
-    build:
-      dockerfile: dockerfiles/Dockerfile.audit
-      context: .
+  mongodb:
+    image: mongo:latest
     volumes:
-      - ./:/scan
+      - "dbdata_mongodb:/data/db"
+    environment:
+      MONGO_INITDB_ROOT_USERNAME:
+      MONGO_INITDB_ROOT_PASSWORD:
+    restart: always
 
   test-management:
     build:
       dockerfile: dockerfiles/Dockerfile.tms
       context: .
+    depends_on:
+      - mongodb
     environment:
       BITBAR_ACCOUNT_MAX:
+      MONGODB_HOST: mongo
+      MONGODB_DB: test-reports
+      MONGODB_USER:
+      MONGODB_PASS:
+    links:
+      - "mongodb:mongo"
     ports:
       - "9340:9340"
+    restart: always
+
+  metabase:
+    image: metabase/metabase
+    depends_on:
+      - mongodb
+    environment:
+      MB_DB_FILE: /metabase-data/metabase.db
+    links:
+      - "mongodb:mongo"
+    volumes:
+      - "dbdata_metabase:/metabase-data"
+    ports:
+      - "3000:3000"
```

Comment:
Do we also need metabase exposed to the world or could people just SSH tunnel it out if they need it?